components/event-block/EventBlockColofon.vue

```diff
@@ -0,0 +1,152 @@
+<template functional>
```

Comment:
**Praise**: nice! I've never used `<template functional>` but I think we can use this much more often in projects. 💪